CHANGELOG.md

```diff
@@ -35,3 +35,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 <!-- markdown-link-check-disable -->
 
 # Changelog
+
+## [Unreleased]
+
+### State Machine Breaking
+
+* (x/farming) [\#305](https://github.com/cosmosquad-labs/squad/pull/305) Time-based queued staking and new UnharvestedRewards struct
```

Comment:
Could you add comments about added, changed API endpoints?

Reply:
Sure, I'll add it.